tests/3d_examples/test_3d_poiseuille_flow_shell/poiseuille_flow_shell.cpp

```diff
@@ -259,10 +259,6 @@ int main()
      */
     water_block.addBodyStateForRecording<int>("Indicator");
     water_block.addBodyStateForRecording<Real>("Pressure");
-    shell_boundary.addBodyStateForRecording<double>("Mass");
-    shell_boundary.addBodyStateForRecording<double>("Density");
-    shell_boundary.addBodyStateForRecording<double>("VolumetricMeasure");
-    shell_boundary.addBodyStateForRecording<double>("Thickness");
```

Comment:
It failed with float because you make an explicit use of `double`, you need to use `Real`

Reply:
I see, thanks a lot!